### ISSUE TITLE
Add proper position uncertainties

### ISSUE THREAD

**Issue by dlakaplan** [mattpitkin/psrqpy#59]
Right now the position uncertainties are hard to parse since they depend on the precision of the sexadecimal position.  I've asked ATNF to add an error on RAJD/DECJD (or the other decimal coordinates), but meanwhile would it be possible to add the position precision/error parsing into this package?

**Comment by mattpitkin** [comment]
Hi @dlakaplan, thanks for the suggestion. Can you take a look #60 and see if it does what you need?
